explore-analyze/visualize/charts/area-charts.md

```diff
@@ -0,0 +1,168 @@
+
+4. Optionally, set **Rank by** for the breakdown dimension to control stacking order.
+
+### Compare current versus previous period with time shift [area-timeshift]
+
+In Area charts, you can enable time shift to compare different periods and identify deltas.
+
+![Example Lens area chart compare periods](../../images/kibana-area-compare-periods.png)
+
+1. Create an area chart with a time-based **Horizontal axis** and your main metric on the **Vertical axis**, for example, `bytes`.
+2. From the three-dot menu in the upper-left of the layer panel, select **Duplicate layer**.
+3. From the duplicated layer, open the **Advanced** of the **Vertical axis** and set **Time shift** to `1w`. 
+   Check also [Compare differences over time](../lens.md#compare-data-with-time-offsets) for more details.
+4. Use a different color and set **Stacking** to **None** so areas overlay rather than stack.
+5. Update the legend to show “Current” and “Previous”.
+
+::::{tip}
+You can also compute the relative change using a formula, for example:  
+`(average(bytes) - average(bytes, shift='1w')) / average(bytes, shift='1w')`
+:::: 
+
+## Area chart settings [area-chart-settings]
+
+Customize your area chart to match the information you need and how you want it displayed.
+
+### Horizontal axis settings [horizontal-axis-settings]
+
+**Data**
+:   **Functions**: Allow you to group your data. For example, you can use `Date histogram` to group data points into time-based buckets, or `Filters` to divide values into predefined subsets.
+:   **Fields**: Determine which field from your data will be used for the horizontal axis.
+:   **Minimum interval**: Determine the smallest time bucket that your data will be grouped into (for example, seconds, minutes, hours, days).
+
+**Appearance**
+:   **Name**: By default, the chart uses the function or formula as title. It's a best practice to customize this with a meaningful title.
+
+### Vertical axis settings [vertical-axis-settings]
+
+**Data**
+:   To represent the metrics or values you want to visualize, you can use quick functions like `Average`, `Count`, `Percentile`, `Counter rate`, or create custom calculations with formulas. Refer to [](/explore-analyze/visualize/lens.md#lens-formulas) for examples, or to the {icon}`documentation` **Formula reference** available from Lens.
+
+    :::{include} ../../_snippets/area-vertical-axis-advanced-settings.md
+    :::
+
+**Appearance**
+:   Configure series-level options, including:
+   - **Name**: Customize the series label.
+   - **Value format**: Control how numeric values are displayed on your vertical axis and in tooltips.
+   - **Series color**: Determine the color of your data series in the visualization.
+   - **Axis side**: Determine which side of the chart the vertical axis appears on.
+
+### Breakdown settings [breakdown-settings]
+
+You can optionally split your series by a categorical field to create multiple stacked or overlapping areas. 
```

Comment:
Let's add an example here.

Reply:
We should be more granular/exhaustive with the whole settings reference part here. Similar to my comments on the Line charts page and to what we delivered with the Metrics page